sources/LLVMSharp/DataLayout.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft and Contributors. All rights reserved. Licensed under the University of Illinois/NCSA Open Source License. See LICENSE.txt in the project root for license information.
+
+using System;
+using LLVMSharp.Interop;
+
+namespace LLVMSharp
+{
+    public sealed class DataLayout : IEquatable<DataLayout>
+    {
+        public DataLayout(ReadOnlySpan<char> stringRep)
+        {
+            Handle = LLVMTargetDataRef.FromStringRepresentation(stringRep);
+        }
+
+        public LLVMTargetDataRef Handle { get; }
+
+        public StructLayout CreateStructLayout(StructType structType)
+        {
+            return new StructLayout(this, structType);
+        }
+
+        public ulong TypeSizeInBits(Type type)
```

Comment:
Likewise, prefix with `Get` and similar for others below where it is a non-property and the C++ name is `get*`
```suggestion
        public ulong GetTypeSizeInBits(Type type)
```